pkgs/tools/networking/smartdns/default.nix

```diff
@@ -1,34 +1,32 @@
-{ lib, stdenv, fetchFromGitHub, openssl }:
+{ lib, stdenv, fetchFromGitHub, openssl, testVersion, smartdns }:
```

Comment:
I am not familiar with testVersion, and it seems to me weird to have self reference here.

Reply:
https://github.com/NixOS/nixpkgs/blob/master/pkgs/build-support/trivial-builders.nix#L810

Currently there's no better way to do selfreferences (better ways are being worked on). if you'd like to learn more then I recommend checking out the pr that added testVersion

Reply:
this would fix it https://github.com/NixOS/nixpkgs/pull/119942